roles/container-engine/containerd/tasks/main.yml

```diff
@@ -129,4 +129,13 @@
     - not is_ostree
     - not runc_stat.stat.exists
 
+- name: Ensure latest version of libseccomp installed  # noqa 403
+  package:
+    name: libseccomp
+    state: latest
+  when:
+    - ansible_distribution == "CentOS"
```

Comment:
but what about the RedHat or Fedora ?
do they need this package too?

Reply:
This is also copied and pasted from the CRI-O code: but I can say I haven't had this problem when testing with Fedora. Some other issues blocked me from investigating RHEL8, but I'll see if I can get past those and let you know.

Reply:
@LuckySB Now that I've managed to get a developer subscription and test: I can confirm that RHEL8 and Fedora do not seem to need this package explicitly installed.

Seems my RHEL issues had been encountered before and fixed but there was a typo, I'll raise another PR :)

Reply:
chrony depends on libseccomp, that is why, please make it for all RedHat family (EL7/EL8)